### ISSUE TITLE
Notification tray support

### ISSUE THREAD

**Issue by joewood9364** [progzone122/warp-cloudflare-gui-rust#6]
Thanks for making this. Just suggesting a simple notification icon to allow quick toggling between on and off states.

**Comment by progzone122** [comment]
Good idea, postponed for now because the programme code will be rewritten (#2)
